docs/install/quick-start.rst

```diff
@@ -0,0 +1,36 @@
+.. meta::
+  :description: Install rocDecode
+  :keywords: install, rocDecode, AMD, ROCm
+
+
+Installation
```

Comment:
This item should be **Quick start installation**
Same as TOC entry
![image](https://github.com/ROCm/hipfort/assets/164940351/99e18cb8-1d68-47e0-af14-2204297eddf2)